jest.config.ts

```diff
@@ -1,11 +1,58 @@
-/** @type {import('ts-jest').JestConfigWithTsJest} */
-module.exports = {
+import { getSDKs } from './harness/helpers'
+
+const commonConfig = {
     preset: '@trendyol/jest-testcontainers',
     transform: {
         '^.+\\.tsx?$': ['ts-jest', {}]
     },
     globalSetup: './jest-global.ts',
     testEnvironment: './jest-environment.ts',
     setupFilesAfterEnv: ['./jest-setup.js'],
+}
+
+const projects = [
```

Comment:
new Jest projects are defined here, that let us change the jest config on a per-project basis.